rollup.config-es6.js

```diff
@@ -0,0 +1,55 @@
+const commonjs = require('rollup-plugin-commonjs');
+const nodeResolve = require('rollup-plugin-node-resolve');
+const replace = require('rollup-plugin-replace');
+const babel = require('rollup-plugin-babel');
+const uglify = require('rollup-plugin-uglify');
+const { minify } = require('uglify-es');
+
+const DEV = !!JSON.parse(process.env.BUILD_DEV || 'true');
+const DEMO = !!JSON.parse(process.env.BUILD_DEMO || 'false');
+
+const plugins = [
+  babel({
+    babelrc: false,
+    presets: [
+      [
+        'es2016',
```

Comment:
I wonder if we can just remove any preset completely?

If we want to use a preset, we should use browser-env.

Reply:
Using `babel-preset-env` which takes in a browserlist query.

We can support latest 2 versions of Chrome, Firefox and Safari?

http://browserl.ist/?q=last+2+firefox+versions%2C+last+2+chrome+versions%2C+last+2+safari+versions

Reply:
https://github.com/babel/babel/tree/master/experimental/babel-preset-env

Reply:
Removed completely